src/sanitizer.py

```diff
@@ -0,0 +1,21 @@
+"""Helper functions to sanitize/unsanitize str, List[str]"""
+
+# (fixme) remove once we move to gremlin driver
+
+from typing import Union, List
+from urllib.parse import quote, unquote
+
+def sanitize(string: Union[str, List[str]]) -> str:
```

Comment:
For my knowledge, if you don't do sanitize/unsanitize then is it failing while querying into graph db ? and to overcome we have aded this feature, is my understanding correct ?

Reply:
It is not failing. Since we generate gremlin query string by appending data from user input, it is better to sanitize it to avoid query injection.

Reply:
got it, thanks